policyengine_us/variables/gov/ssa/ssi/eligibility/income/ssi_countable_income.py

```diff
@@ -14,13 +14,23 @@ class ssi_countable_income(Variable):
 
     def formula(person, period, parameters):
         earned_income = person("ssi_marital_earned_income", period)
+        blind_disabled_working_student_income = person(
+            "ssi_blind_or_disabled_working_student", period
+        )
+        capped_earned_income = max_(
+            earned_income - blind_disabled_working_student_income, 0
+        )
+
         unearned_income = person("ssi_marital_unearned_income", period)
         parental_income_deemed_as_unearned_income = person(
             "ssi_unearned_income_deemed_from_ineligible_parent", period
         )
+        blind_disabled_working_student_income = person(
+            "ssi_blind_or_disabled_working_student", period
+        )
 
         personal_income = _apply_ssi_exclusions(
-            earned_income,
+            capped_earned_income,
```

Comment:
I think this input has to be "earned_income" so lets use "pre_reduction_earned_income" and "earned_income" for lines 16 and 20